### ISSUE TITLE
Commit 2efb59e broke lambda invocation on Mac

### ISSUE THREAD

**Issue by troussan** [localstack/localstack#1609]
Localstack 0.10.3.2 fails to import lambda function python module on Mac:

{
E             "errorType": "Runtime.ImportModuleError",
E             "errorMessage": "Unable to import module 'get_assessment/handler': No module named 'get_assessment/handler'"
E           } Traceback (most recent call last):
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_api.py", line 338, in run_lambda
E               event, context=context, version=version, asynchronous=asynchronous)
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 83, in execute
E               return do_execute()
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 68, in do_execute
E               result, log_output = self._execute(func_arn, func_details, event, context, version)
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 229, in _execute
E               result, log_output = self.run_lambda_executor(cmd, stdin, environment)
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 149, in run_lambda_executor
E               (return_code, log_output))
E      

It works fine with 0.10.3.1. The error is caused by the line 55-57 move down to the end of file licalstack/config.py:
```
 if TMP_FOLDER.startswith('/var/folders/') and os.path.exists('/private%s' % TMP_FOLDER):
    TMP_FOLDER = '/private%s' % TMP_FOLDER
```
These lines setup Mac TMP folder. Why these lines were moved?

**Comment by troussan** [comment]
@whummer

**Comment by whummer** [comment]
Fixed in #1610 - thanks
